Add example telemetry report to documentation (#4861)
FerretDB/FerretDB: website/docs/telemetry.md

The same information is always saved in a `telemetry.json` file
in the [state directory](configuration/flags.md#miscellaneous), making it easy to inspect.

> For example, this is the content of this file at the time of writing on the Beacon service itself
> (it, of course, uses FerretDB to store data):

Reviewer: "...content of this file at the time of writing on the Beacon service" is a bit confusing. I'm guessing the file is `temetry.json` but don't see how the user will relate to the "at the time of writing on the Beacon service". Maybe something like this:

```suggestion
For example, this is the content of a `telemetry.json` file as recorded by the Beacon service (which, of course, uses FerretDB to store data)
```
